news/2020-02-20-results-for-monthly-beatmapping-contest-january.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "Results for Monthly Beatmapping Contest: January"
+date: 2020-02-20 8:00:00 +0000
+---
+
+The votes are in! Find out the champions of January's Monthly Beatmapping Contest!
+
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+In case you're out of the loop, the **Monthly Beatmapping Contest** (*MBC*) has been revived for a new run in 2020. Read the contest schedule and details on the [MBC wiki article](/wiki/Contests/Monthly_Beatmapping_Contest). Got everything? Good. Onto the results!
+
+## Winners
+
+Votes for all finalists can be found on the [contest's listing](https://osu.ppy.sh/community/contests/84).
+
+**Community vote winner:** [**MYLK - Kiradoki Harmony**](https://osu.ppy.sh/beatmapsets/1111940#osu/2323164) by [**melon boy**](https://osu.ppy.sh/users/3053382) (previously known as Fuccho)
+
+Receiving a towering **78** votes, melon boy's submission climbed to the top of the community vote poll and is crowned the #1 entry in this month's contest! This beatmap features tons of 145 BPM stream jumps with a large circle size, making for a difficult yet manageable alternating-focused experience!
```

Comment:
tenses dont match up in first sentence but I thought about this for too long and now I'm not sure what to change to fix it. do u see what i mean at least pls help